sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -164,6 +166,7 @@ case class CatalogTable(
     stats: Option[Statistics] = None,
     viewOriginalText: Option[String] = None,
     viewText: Option[String] = None,
+    currentDatabase: Option[String] = None,
```

Comment:
ok. I think we need to change this name. People will be confused with this variable and identifier's database name part. Also, it will be good to make the name reflect that it is view-specific.